Makefile

```diff
@@ -0,0 +1,17 @@
+clean:
+	@rm -rf `find . -name __pycache__`
+	@rm -f `find . -type f -name '*.py[co]' `
+	@make -C docs clean
+
+doc:
+	@make -C docs html SPHINXOPTS="-W -E"
+	@echo "open file://`pwd`/docs/_build/html/index.html"
```

Comment:
I think this action is redundant and not friendly. I may not have a XServer at all while I'm working on docs (just want to make sure that they builds right) or I  already have a tab with docs which I could simply reload instead of generate dozens of them for each `make docs` call.

Reply:
Let's keep it.
I use this printout in aiohttp very often.
You should not open a docs file every time but the hint is useful if you run the command first time.
Extra line in console output doesn't harm.